js/modules/backup.js

```diff
@@ -620,7 +629,7 @@ async function exportConversation(db, conversation, options) {
         const jsonString = JSON.stringify(stringify(message));
         stream.write(jsonString);
 
-        if (attachments && attachments.length) {
+        if (attachments && attachments.length > 0) {
```

Comment:
What, you don't like my truthiness shorthand here? What do you have against my C heritage? ;0)

Reply:
Vintage™!